apps/www/_blog/2023-11-21-oauth2-login-python-flask-apps.mdx

```diff
@@ -0,0 +1,140 @@
+---
+title: 'Implementing GitHub OAuth2 login for Python Flask app'
+description: Learn how to implement OAuth in your Python Flask applications.
+categories:
+  - engineering
+tags:
+  - python
+  - auth
+date: '2023-11-21'
+toc_depth: 3
+author: andrew_smith
+image: oauth2-login-python-flask-apps/OG-image.jpg
+thumb: oauth2-login-python-flask-apps/OG-image.jpg
+---
+
+OAuth is an open standard protocol facilitating secure authorization and authentication for third-party applications to access user resources without obtaining their credentials directly. The OAuth 2.0 version involves users granting permission, authorization servers issuing access tokens, and resource servers verifying tokens to enable controlled access across various web services and APIs.
```

Comment:
```suggestion
In this guide we'll learn how to quickly build an OAuth2.0 integration into a simple Flask app using Supabase-py. This will enable your users to login to your web app using their GitHub account.
```